scripts/homestead.rb

```diff
@@ -267,7 +267,7 @@ def self.configure(config, settings)
           end
 
           s.path = script_dir + "/serve-#{type}.sh"
-          s.args = [site['map'], site['to'], site['port'] ||= http_port, site['ssl'] ||= https_port, site['php'] ||= '7.3', params ||= '', site['zray'] ||= 'false', site['exec'] ||= 'false', headers ||= '', rewrites ||= '']
+          s.args = [site['map'], site['to'], site['port'] ||= http_port, site['ssl'] ||= https_port, site['php'] ||= '7.3', params ||= '', site['zray'] ||= 'false', site['xhgui'] ||= '', site['exec'] ||= 'false', headers ||= '', rewrites ||= '']
```

Comment:
This is starting to get pretty tricky, and was a lot of copy-pasta for the shell scripts. I think this could be refactored (the nginx templates are substantially similar), but the arguments are not 100% consistent between each one. I've left this as is, thinking a followup PR can clean this and the shell scripts up either by extracting some functions in bash, or perhaps going all in and converting it to be a RoboFile to run with https://robo.li.